File: python/marks_average.py
def main():
    mark1 = int(input("Enter the first mark:\n"))
    mark2 = int(input("Enter the second mark:\n"))
    mark3 = int(input("Enter the third mark:\n"))

    average = (mark1 + mark2 + mark3) / 3.0
    rounded_average = round(average)

    if rounded_average >= 40:
        print("PASS")
    else:
        print("FAIL")


if __name__ == "__main__":
    main()

File: javascript/MarksAverage.js
const readline = require("readline");

function askFactory() {
  const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
  });

  const ask = (question) =>
    new Promise((resolve) => {
      rl.question(question, (answer) => resolve(answer));
    });

  return { rl, ask };
}

async function main() {
  const { rl, ask } = askFactory();

  const mark1 = Number(await ask("Enter the first mark:\n"));
  const mark2 = Number(await ask("Enter the second mark:\n"));
  const mark3 = Number(await ask("Enter the third mark:\n"));

  const average = (mark1 + mark2 + mark3) / 3.0;
  const roundedAverage = Math.round(average);

  if (roundedAverage >= 40) {
    console.log("PASS");
  } else {
    console.log("FAIL");
  }

  rl.close();
}

main();
